timesketch/lib/aggregators/__init__.py

```diff
@@ -17,3 +17,4 @@
 from timesketch.lib.aggregators import bucket
 from timesketch.lib.aggregators import feed
 from timesketch.lib.aggregators import term
+from timesketch.lib.aggregators import vega
```

Comment:
Is this an aggregation or chart? i.e. should it be in the charts/ dir instead?

Reply:
no, this is an aggregator, since you need to be able to add it into stories ,etc... and you want it to be displayed like other aggregators, if you would have this as a chart, then it would need to be added as part of another aggregator, and that would not work.

Reply:
ack